--- BUILD
+++ BUILD
@@ -8,6 +8,7 @@ exports_files(["filter_swig_code.py", "build_integrated.py",
 	       "replace_swig_import_helper.py"])
 
 load("//:variables.bzl", "VM_ENABLED_DEFINES")
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
 
 config_setting(
     name = "benchmark",

--- MODULE.bazel
+++ MODULE.bazel
@@ -1,1 +1,2 @@
 module(name="exaudfclient_base", version = "1.0")
+bazel_dep(name = "rules_cc", version = "0.2.22")

--- exaudflib/BUILD
+++ exaudflib/BUILD
@@ -4,1 +4,2 @@
 load("//:variables.bzl", "VM_ENABLED_DEFINES")
+load("@rules_cc//cc:defs.bzl", "cc_library")

--- exa_udfclient_binary_common/BUILD
+++ exa_udfclient_binary_common/BUILD
@@ -1,1 +1,2 @@
 package(default_visibility = ["//visibility:public"])
+load("@rules_cc//cc:defs.bzl", "cc_library")
